controllers/imagejob/imagejob_controller.go

```diff
@@ -276,12 +286,35 @@ func (r *Reconciler) handleNewJob(ctx context.Context, imageJob *eraserv1alpha1.
 		return err
 	}
 
+	skipped := 0
+
 	log := log.WithValues("job", imageJob.Name)
 
+nodes:
 	for i := range nodes.Items {
 		log := log.WithValues("node", nodes.Items[i].Name)
 
 		nodeName := nodes.Items[i].Name
+		for _, skipNodesSelector := range skipNodesSelectors {
+			skipLabels, err := labels.Parse(skipNodesSelector)
+			if err != nil {
+				return err
+			}
+
+			log.Info("skipLabels", "skipLabels", skipLabels)
+			log.Info("nodeLabels", "nodeLabels", nodes.Items[i].ObjectMeta.Labels)
```

Comment:
these seem too verbose for Info, should we remove or move debug?

Reply:
Preferable to move it to debug, as it's useful for troubleshooting.

- [ ] I'll look into how to manage verbose logging.

Reply:
Will now only appear when --log-level=debug is set on the pod

Reply:
Shouldn't these be set to `log.Debug`?

Reply:
The `Logger` struct doesn't have a `Debug()` function, but we can put them at a higher level with `log.V(1).Info(...)`, which hides them from pod logs unless `--log-level=debug` flag is provided. I'll do that unless there's a better way.